src/nav/patterns.yml

```diff
@@ -8,3 +8,5 @@
       items:
         - id: /patterns/tables/basic-formatting
           title: Basic formatting
+    - title: Rich-text editor
+      id: /patterns/rich-text-editor
```

Comment:
minor nitpick to fix back to alphabetical order. ˙ ͜ʟ˙

Reply:
![image](https://user-images.githubusercontent.com/773721/187887801-6c544df7-570b-4983-b4e2-28e23c1f092e.png)

Pushed an update, should be fixed now. Thanks for flagging this, missed it 🙈